File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopLocationResponseParam.java
package com.cakk.domain.mysql.dto.param.shop;

import java.util.Set;

public record CakeShopLocationResponseParam(
	Long cakeShopId,
	String thumbnailUrl,
	String cakeShopName,
	String cakeShopBio,
	Set<String> cakeImageUrls,
	Double longitude,
	Double latitude
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopLocationResponseParam.kt
package com.cakk.domain.mysql.dto.param.shop

data class CakeShopLocationResponseParam(
	val cakeShopId: Long,
	val thumbnailUrl: String,
	val cakeShopName: String,
	val cakeShopBio: String,
	val cakeImageUrls: Set<String>,
	val longitude: Double,
	val latitude: Double
)
